src/System.Runtime/tests/System/TypeTests.netcoreapp.cs

```diff
@@ -36,5 +36,42 @@ public void IsSZArray(Type type, bool expected)
         {
             Assert.Equal(expected, type.IsSZArray);
         }
+
+        private static IEnumerable<object[]> VariableBoundArrayOrNotTypes()
+        {
+            // Not arrays
+            yield return new object[] { typeof(void), false };
+            yield return new object[] { typeof(int), false };
+            yield return new object[] { typeof(int[]).MakeByRefType(), false };
+            yield return new object[] { typeof(TypeTests), false };
+            yield return new object[] { Type.GetType("System.Int32"), false };
+            yield return new object[] { typeof(Outside<int>.Inside<string>), false };
+
+            // Arrays, but SZ arrays
+            yield return new object[] { Type.GetType("System.Int32[]"), false };
+            yield return new object[] { typeof(int[]), false };
```

Comment:
> Remove duplicates from the IsSZArray tests too?

I wouldn't consider it mandatory for this PR but if you'd like to make them consistent, sure. It might be better just to use the same testdata for both tests.